dotnetcore3.1/cookiecutter-aws-sam-quick-start-web-dotnet/{{cookiecutter.project_name}}/src/{{cookiecutter.project_name}}/LocalEntryPoint.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace {{cookiecutter.project_name}}
+{
+    /// <summary>
+    /// The Main function can be used to run the ASP.NET Core application locally using the Kestrel webserver.
```

Comment:
Users can test their functions locally by running `sam local invoke` or `sam local start-api`. And other reason is, this might create an environment different then lambda, and lead to some mis-alignment with their actual deployment vs testing locally.

For that reason, I would suggest to remove this file, and again, maybe add a comment on README about how they can invoke their function locally.

Reply:
@mndeveci Done... Can you please check... and feel free to ping me on salck we can go to the review